src/pods/impl.c

```diff
@@ -191,37 +191,39 @@ void macIDToUpper(char macValue[],char macConverted[])
 
 char* get_deviceMAC()
```

Comment:
@Abinesh44-Devi same func is getting used for webconfig feature for pods, is it fine for webconfig as well for pods?

Reply:
Yes